query/parse.go

```diff
@@ -138,7 +140,11 @@ func parseExpr(in []byte) (Q, int, error) {
 		}
 		expr = q
 	case tokLang:
-		expr = &Language{Language: text}
+		canonical, ok := enry.GetLanguageByAlias(text)
+		if !ok {
+			canonical = "UKNNOWN"
```

Comment:
should we instead just evaluate this to `&Const{false}` in this case? IE it won't match any files.